src/reference/asciidoc/jdbc.adoc

```diff
@@ -1085,6 +1085,11 @@ If so, you can specify the `id` to be associated with the `DefaultLockRepository
 Starting with version 5.1.8, the `JdbcLockRegistry` can be configured with the `idleBetweenTries` - a `Duration` to sleep between lock record insert/update executions.
 By default it is `100` milliseconds and in some environments non-leaders pollute connections with data source too often.
 
+Starting with version 5.4, the `RenewableLockRegistry` interface has been introduced and added to `JdbcLockRegistry`.
+The `renewLock()` method must be called during locked process in case of the locked process would be longer than time to live of the lock.
+So the time to live can be highly reduce and deployments can retake a lost lock quickly.
+NB. The lock renewal can be done only if the lock is held by the current thread, so the locked process has to be executed in another thread.
```

Comment:
I'm not sure what is this `NB`  about.
Can we have a more official language in the docs, please?

Reply:
NB (latin Nota Bene - "note well") is commonly used in real (British) English - it is not common in the USA.

Use `NOTE:` instead.

Reply:
Oh, right! Didn't think through. This sentence is really `NOTE:`.

Thank you, Gary!

OK. So, I'm pulling this locally and fixing the latest simple concerns on merge.